Makefile.am

```diff
@@ -6200,6 +6282,10 @@ clean-local: $(CLEAN_LOCAL_HOOKS)
 	rm -f $(abs_srcdir)/hwdb/usb.ids $(abs_srcdir)/hwdb/pci.ids $(abs_srcdir)/hwdb/oui.txt \
 	      $(abs_srcdir)/hwdb/iab.txt
 
+
+installlibLTLIBRARIES = install-libLTLIBRARIES
+$(installlibLTLIBRARIES): install-rootlibexecLTLIBRARIES
```

Comment:
This ensures that libshared is installed before any other libraries. Otherwise, libtool relinking may fail as it cannot find libshared.